packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -340,37 +340,56 @@ static CameraPosition toCameraPosition(Object o) {
     return builder.build();
   }
 
-  static CameraUpdate toCameraUpdate(Object o, float density) {
-    final List<?> data = toList(o);
-    switch (toString(data.get(0))) {
-      case "newCameraPosition":
-        return CameraUpdateFactory.newCameraPosition(toCameraPosition(data.get(1)));
-      case "newLatLng":
-        return CameraUpdateFactory.newLatLng(toLatLng(data.get(1)));
-      case "newLatLngBounds":
-        return CameraUpdateFactory.newLatLngBounds(
-            toLatLngBounds(data.get(1)), toPixels(data.get(2), density));
-      case "newLatLngZoom":
-        return CameraUpdateFactory.newLatLngZoom(toLatLng(data.get(1)), toFloat(data.get(2)));
-      case "scrollBy":
-        return CameraUpdateFactory.scrollBy( //
-            toFractionalPixels(data.get(1), density), //
-            toFractionalPixels(data.get(2), density));
-      case "zoomBy":
-        if (data.size() == 2) {
-          return CameraUpdateFactory.zoomBy(toFloat(data.get(1)));
-        } else {
-          return CameraUpdateFactory.zoomBy(toFloat(data.get(1)), toPoint(data.get(2), density));
```

Comment:
toPoint is also dead code, which cascades to toPixels and toFractionalPixels being dead code as well.

I believe if you edit this file in Android Studio, the "problems" pane will flag anything that is `static` and unused.

Reply:
(As a general note, I would expect that when the Pigeon conversion of `google_maps_flutter_android` is complete, there would be no methods in this file that take an `Object` as their parameter, so it would be good to audit to make sure that dead code is removed, and anything left already has a tracking bug. Per [this comment](https://github.com/flutter/flutter/issues/152926) I'm not sure we have all the remaining work tracked in issues.)